scripts/windows/agent_diagnostics.ps1

```diff
@@ -0,0 +1,87 @@
+﻿# Get OS version and build info
+$UNAME=(Get-WmiObject Win32_OperatingSystem | select Caption, Version, BuildNumber)
+$SYSINFO=$UNAME.Caption
+$BUILD=$UNAME.Version
+$SERVICE='jumpcloud-agent'
+$JCPATH='C:\Program Files\JumpCloud'
+$SERVICEVERSION=(Get-Content -Path (Join-Path -Path $JCPATH -ChildPath Plugins\Contrib\version.txt))
+$JCLOG='C:\Windows\Temp\'
+$STAMP=Get-Date -Format 'yyyyMMddhhmmss'
+$TZONE=([TimeZoneInfo]::Local).DisplayName
+$STATUS=(Get-Service $SERVICE).Status
+$WRITEPATH=$HOME+'\Desktop\'
+$ZIPFILE='jc'+$STAMP
+$OUTPUTFILE=$WRITEPATH+'output.log'
+
+function zipjc {
+    # Take inventory of files to be zipped.
+    $TEMP=Join-Path -Path $(pwd) -ChildPath 'temp'
+    $INVENTORY=(Get-ChildItem -Exclude *.crt, *.key -Recurse $JCPATH).Name
+    if(Test-Path $WRITEPATH\$ZIPFILE) {
+        $zipjc_out=("`t$ZIPFILE exists moving to $ZIPFILE.bak.zip")
+        mv $WRITEPATH\$ZIPFILE.zip $WRITEPATH\jc$STAMP.bak -Force
+        }
+    $zipjc_out=("$(foreach($in in $INVENTORY) { ("`t$in`n") })")
+    Get-ChildItem -Path $JCPATH | % {
+        Copy-Item $_.FullName $TEMP -Recurse -Force -Exclude @("*.crt", "*.key")
+}
+    Compress-Archive -Path $TEMP -DestinationPath $WRITEPATH\$ZIPFILE
+    Remove-Item -Path $TEMP -Recurse
+    return $zipjc_out
+}
+
+function ziplog {
+    # Zip the log files.
+    $LOGFILE="jcagent.log"
```

Comment:
Is there a `jcUpdate.log` on Windows that we should grab as well?

Reply:
There are multiple log files I missed on the first pass. I added a loop to get all jc* and Jump* files from c:\Windows\Temp.